docs/contributing/PrefetchingWithFetchData.md

```diff
@@ -0,0 +1,31 @@
+Earlier, we mentioned the static fetchData method. React Router allows you to call a method before rendering a component. In this case, all of the containers have the opportunity to let the server and client know if you need to fetch data before showing the component.
+To use this method, create an action creator using the Promise middleware and return the result of dispatching that action.
```

Comment:
We can try to make some more spaces here and in other docs. E.g. here, you can add an empty line before `To use this` and after it.

The convention would be to keep paragraphs separate (not line by line)